wordpress-mobile/AztecEditor-iOS
# Allow to use headers without bold/shadow traits

Fixes #1185

This PR adds an option on the library to select the use of bold/shadow on headings formaters configurable for library users.

To test:
 - Start the demo app
 - Check if headers are showing correctly and bold format works correctly
 - Try it again but now with the option `Configuration.headersWithBoldTrait` set to true
 - Check the HTML generated to see if headers don't get extra `strong` elements.
